deepfence_ui/app/scripts/components/common/agent-setup/index.jsx

```diff
@@ -52,29 +56,186 @@ export const AgentSetup = () => {
     }
   }, [state]);
 
+
+
+  const getK8sInstructions = () => {
+    return `helm repo add deepfence https://deepfence-helm-charts.s3.amazonaws.com/threatmapper
+helm repo update
+
+helm install deepfence-agent deepfence/deepfence-agent \\
+  --set managementConsoleUrl=${window.location.host ?? '---CONSOLE-IP---'} \\
+  --set deepfenceKey=${
+    localStorage.getItem('dfApiKey') ?? '---DEEPFENCE-API-KEY---'
+  } \\
+  --set image.tag=${process.env.__PRODUCTVERSION__} \\
+  --set image.clusterAgentImageTag=${process.env.__PRODUCTVERSION__} \\
+  --set clusterName=${clusterName} \\
+  --namespace ${namespace || 'deepfence'} \\
+  ${containerRuntime || '--set {container_runtime}'} \\
```

Comment:
What will be default value for these before selection from option? Means how does it appear on instructions?

Reply:
we show placeholder instructions by default and when you select the options, we show the corresponding instructions